howto/mro.rst:13: [timeout] https://www.phyast.pitt.edu/~micheles/: HTTPSConnectionPool(host='www.phyast.pitt.edu', port=443): Max retries exceeded with url: /~micheles/ (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f661d52a510>, 'Connection to www.phyast.pitt.edu timed out. (connect timeout=30)'))
library/unittest.rst:67: [timeout] http://lists.idyll.org/listinfo/testing-in-python: HTTPConnectionPool(host='lists.idyll.org', port=80): Max retries exceeded with url: /listinfo/testing-in-python (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f661abb5010>, 'Connection to lists.idyll.org timed out. (connect timeout=30)'))
